litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowGraphs/ScheduleAndRunStats/style.ts

```diff
@@ -9,6 +9,7 @@ const useStyles = makeStyles((theme: Theme) => ({
   graphContainer: {
     width: '900px',
     height: '320px',
+    margin: theme.spacing(2.5, 0, 0, -6.25),
```

Comment:
avoid negative margins

Reply:
This is required for graph containers...I had this discussion with the others already @ishangupta-ds